glean-core/android/src/test/java/mozilla/telemetry/glean/pings/CustomPingTest.kt

```diff
@@ -221,4 +226,54 @@ class CustomPingTest {
         assertEquals("ui.click", "$category.$name")
         assertEquals(0, event.getLong("timestamp"))
     }
+
+    @Test
+    fun `custom pings can be disabled remotely`() {
+        delayMetricsPing(context)
+        resetGlean(
+            context,
+            Glean.configuration.copy(
+                serverEndpoint = "http://" + server.hostName + ":" + server.port,
+            ),
+            clearStores = true,
+            uploadEnabled = true,
+        )
+
+        // Define a new custom ping inline.
+        val customPing = PingType<NoReasonCodes>(
+            name = "custom-ping",
+            includeClientId = true,
+            sendIfEmpty = true,
+            preciseTimestamps = true,
+            includeInfoSections = true,
+            reasonCodes = emptyList(),
+            enabled = true,
```

Comment:
I was about to be confused that the order didn't need changing here, but I forgot that Kotlin has named args.

Reply:
Yep, this didn't break when I built so it didn't register. I'll update for consistency.